.github/workflows/build.yaml

```diff
@@ -152,20 +152,22 @@ jobs:
       date: ${{ inputs.date }}
       package-name: cuopt_server
       package-type: python
-  #docs-build:
-  #  if: inputs.build_type == 'nightly' || github.ref_type == 'branch'
-  #  needs: [python-build]
-  #  secrets: inherit
-  #  uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-25.08
-  #  with:
-  #    build_type: ${{ inputs.build_type || 'branch' }}
-  #    node_type: "gpu-l4-latest-1"
-  #    sha: ${{ inputs.sha }}
-  #    branch: ${{ inputs.branch }}
-  #    date: ${{ inputs.date }}
-  #    arch: amd64
-  #    container_image: rapidsai/ci-conda:cuda11.8.0-ubuntu22.04-py3.10
-  #    run_script: ci/build_docs.sh
+  docs-build:
+    if: inputs.build_type == 'nightly' || github.ref_type == 'branch'
```

Comment:
This will (I think) prevent re-building the docs when a new tag is pushed. Is that what you want?

If you're not sure why this is here and it was just copied from somewhere else, I recommend just removing it.